recipes/onnxruntime/all/conanfile.py

```diff
@@ -27,11 +27,13 @@ class OnnxRuntimeConan(ConanFile):
         "shared": [True, False],
         "fPIC": [True, False],
         "with_xnnpack": [True, False],
+        "onnx_no_static_registration": [True, False],
```

Comment:
```suggestion
```
I woudn't add a recipe option because (that I know) `onnxruntime` doesn't have a CMake knob to disable it's own registration.

So the best way to model this is to not add a recipe option, and force `onnx` to be compiled with `onnx:disable_static_registration=True`.